SPECS/ocaml-fileutils/ocaml-fileutils.spec

```diff
@@ -1,27 +1,26 @@
 Summary:        OCaml library for common file and filename operations
 Name:           ocaml-fileutils
-Version:        0.5.2
-Release:        18%{?dist}
+Version:        0.6.4
+Release:        1%{?dist}
 License:        LGPLv2 WITH exceptions
 Vendor:         Microsoft Corporation
 Distribution:   Azure Linux
 URL:            https://github.com/gildor478/ocaml-fileutils
 # NOTE: the "_2" suffix was added to avoid conflicts with an older source tarball for the same version of the sources.
 #       Please remove it during a version update.
-Source0:        https://github.com/gildor478/ocaml-fileutils/archive/refs/tags/%{version}.tar.gz#/%{name}_2-%{version}.tar.gz
-# Set of files from previous sources location allowing us to drop dependency on "ocaml-oasis-devel", which is no longer available.
-# Previous sources used to be available under http://forge.ocamlcore.org/frs/download.php/1695/ocaml-fileutils-0.5.2.tar.gz.
-# Currently still available in Fedora's SRPMs.
-Source1:        %{name}-old-sources-build-files.tar.gz
-Patch1:         fileutils-0.5.2-fix-bytes.patch
-# Use ounit2.
-Patch2:         ocaml-fileutils-0.5.2-ounit2.patch
-
-BuildRequires:  ocaml >= 4.00.1
-BuildRequires:  ocaml-findlib-devel >= 1.3.3-3
-BuildRequires:  ocaml-ocamlbuild
-BuildRequires:  ocaml-ocamldoc
-BuildRequires:  ocaml-ounit-devel
+Source0:        %{url}/archive/refs/tags/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
+# Fedora does not need the stdlib-shims or seq forward compatibility packages
+Patch0:         ocaml-fileutils-0.6.4-forward-compat.patch
+# Given two distinct files with identical contents, the cmp function evaluates
+# to "Some x", where x is the size of each file, instead of None.  This breaks
+# the ocaml-gettext tests, which expect None in that case.
+Patch1:         ocaml-fileutils-0.6.4-cmp.patch
+
+BuildRequires:  ocaml >= 5.1.1
+BuildRequires:  ocaml-dune >= 1.11.0
+%if 0%{?fedora} || 0%{?rhel} <= 6
```

Comment:
Since we're taking this spec from Fedora, it might be wise to keep the Fedora-specific spec paths enabled in Azure Linux as well. You could achieve that by adding `|| %{?azl}` here and elsewhere where `%if 0%{?fedora}` is present.

Reply:
Added it